doc/deployment/connect-to-cluster.md

```diff
@@ -0,0 +1,187 @@
+store buckets located in the same region.
+
+For more information about Pachyderm contexts, see
+[Manage Cluster Access](../managing_pachyderm/connect-to-cluster.html)
+
+To connect by using a Pachyderm context, complete the following
+steps:
+
+1. Get the current context:
+
+   ```bash
+   $ pachctl config get active-context
+   ```
+
+   If no IP address is set up for this cluster, you get the following
+   output:
+
+   ```bash
+   $ pachctl config get context <name>
+   {
+
+   }
+   ```
+
+1. Set up `pachd_address`:
+
+   ```bash
+   $ pachctl config update context <name> --pachd-address <host:port>
+   ```
+
+   **Example:**
+
+   ```bash
+   $ pachctl config update context local --pachd-address 192.168.1.15:30650
+   ```
+
+   **Note:** The `pachd` port is `30650` or `650`.
+
+1. Verify that the configuration has been updated:
+
+   ```bash
+   $ pachctl config get context local
+   {
+      "pachd_address": "192.168.1.15:30650"
+   }
+   ```
+
+## Connect by Using Port-Forwarding
+
+The Pachyderm port-forwarding is the simplest way to enable cluster access
+and test basic Pachyderm functionality. Pachyderm automatically starts
+port-forwarding from `pachctl` to `pachd`. Therefore, the traffic
+from the local machine goes to the `pachd` endpoint through the
+Kubernetes API. However, to open a persistent tunnel to other ports, including
+the Pachyderm dashboard, git and authentication hooks, the built-in HTTP
+file API, and other, you need to run port-forwarding explicitly.
+
+Also, if you are connecting with port-forward, you are using the `0.0.0.0`.
+Therefore, if you are using a proxy, it needs to handle that appropriately.
+
+Although port-forwarding is convenient for testing, for production
+environments, this connection might be too slow and unreliable.
+The speed of the port-forwarded traffic is limited to `1 MB/s`.
+Therefore, if you experience high latency with `put file` and
+`get file` operations, or if you anticipate high throughput
+in your Pachyderm environment, you need to enable ingress access
+to your Kubernetes cluster. Each cloud provider has its own
+requirements and procedures for enabling ingress controller and
+load balancing traffic at the application layer for better performance.
+
+Remember that exposing your traffic through a public ingress might
+create a security issue. Therefore, if you expose your `pachd` endpoint,
+you need to make that you take steps to protect the endpoint and
+traffic against common container security threats. Port-forwarding
+might be an alternative which might provide satisfactory performance
+if you place the data that is consumed by your pipeline in object
+store buckets located in the same region.
+
+<!--Add a link to the section that describes the above-->
+
+To enable port-forwarding, complete the following steps:
+
+1. Open a new terminal window
+1. Run:
+
+   ```bash
+   pachctl port-forward
+   ```
+
+   This command does not stop unless you manually interrupt it.
```

Comment:
Add

You can run pachctl commands from another window.  If you have trouble with any of them completing, check to see if `kubectl`'s port forwarding has had issues, preventing `pachctl port-forward` from running properly.